chrome/Dockerfile

```diff
@@ -1,24 +1,19 @@
-FROM ubuntu:latest
+FROM ubuntu:jammy
 
 ARG VERSION
 ARG DEBIAN_FRONTEND=noninteractive
 ENV DBUS_SESSION_BUS_ADDRESS disabled:
 ENV RD_PORT=9222
 
-RUN apt-get update \
-    # https://github.com/phusion/baseimage-docker/issues/319
-    && apt-get install --yes apt-utils 2>&1 | grep -v "debconf: delaying package configuration, since apt-utils is not installed" \
-    && apt-get install -y wget gnupg \
-    && wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | gpg --dearmor -o /usr/share/keyrings/googlechrome-linux-keyring.gpg \
-    && sh -c 'echo "deb [arch=amd64 signed-by=/usr/share/keyrings/googlechrome-linux-keyring.gpg] https://dl-ssl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list' \
-    && wget --no-verbose -O /tmp/chrome.deb https://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_${VERSION}_amd64.deb \
-    && apt-get update \
-    && apt-get install -y /tmp/chrome.deb fonts-ipafont-gothic fonts-wqy-zenhei fonts-thai-tlwg fonts-khmeros fonts-kacst fonts-freefont-ttf libxss1 dumb-init socat dbus dbus-x11 \
-      --no-install-recommends \
-    && service dbus start \
-    && rm -rf /var/lib/apt/lists/* \
-    && groupadd -r chrome && useradd --create-home -rm -g chrome -G audio,video chrome
-
+RUN apt-get update
+RUN apt-get install -y wget gnupg
+RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | gpg --dearmor -o /usr/share/keyrings/googlechrome-linux-keyring.gpg
+RUN sh -c 'echo "deb [arch=amd64 signed-by=/usr/share/keyrings/googlechrome-linux-keyring.gpg] https://dl-ssl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list'
+RUN apt-get update
+RUN apt-get install google-chrome-stable=${VERSION}
```

Comment:
Google PPA maintains only the latest version for each channel (https://www.ubuntuupdates.org/pm/google-chrome-stable), which can lead to inconsistent and non-reproducible builds

Reply:
We can ignore the change and use the following approach:
- Query chrome version from the google PPA (using new docker image chrome/get_version.Dockerfile).
- Download chrome as .deb package and install it as we did before (revert chrome/Dockerfile).
WDYT?

Reply:
@anton7c3 it is better to use the dedicated API to fetch the latest version: https://github.com/GoogleChromeLabs/chrome-for-testing#json-api-endpoints.

Reply:
After a conversation, I've just changed the version to download.
And pinned the base ubuntu image from `latest` to `bionic`.